src/overlays/actors/ovl_Oceff_Wipe/z_oceff_wipe.h

```diff
@@ -3,12 +3,13 @@
 
 #include <ultra64.h>
 #include <global.h>
+#include <vt.h>
```

Comment:
Include vt.h in the C file instead since other files including this header may not want vt.h too.